system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts

```diff
@@ -0,0 +1,15 @@
+### Cinder Alerts ###
+
+ALERT OpenstackCinderVolumeStuck
+  IF (sum(openstack_stuck_volumes_count_gauge)) by (region,host) *60) > 0
```

Comment:
Hi @rajivmucheli, there are 2 brackets behind `..gauge))`. Remove one. Please check the expression in prometheus beforehand. Also`region` in the by clause is optional. 
I couldn't find the metric you're trying to alert on. Is it supposed to be there yet?